lib/modules/datasource/docker/index.spec.ts

```diff
@@ -48,26 +48,10 @@ describe('modules/datasource/docker/index', () => {
     });
     hostRules.hosts.mockReturnValue([]);
     delete process.env.RENOVATE_X_DOCKER_MAX_PAGES;
-    delete process.env.RENOVATE_X_DOCKER_HUB_TAGS;
+    delete process.env.RENOVATE_X_DOCKER_HUB_TAGS_DISABLE;
   });
 
   describe('getDigest', () => {
-    it('returns null if no token', async () => {
```

Comment:
I couldn't get this to stop calling hub.docker.com, but it wasn't touching any untested code paths